### PR TITLE
CSVS-101 Mapper was used instead of JsonSerializer in EventHandler

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

**Comment by MedvedevBW** [comment]
@placccebo

File: src/main/scala/com/bwsw/cloudstack/vault/server/Components.scala [hunk 1]
```diff
@@ -77,9 +77,11 @@ class Components(settings: Components.Settings) {
     settings.cloudStackVaultControllerSettings
   )
 
+  //event handling
   lazy val consumer = new Consumer[String, String] (
     settings.consumerSettings
   )
+  lazy val eventMapper = new JsonMapper(ignoreUnknownProperties = true)
 
   def close(): Unit = {
```
Comment (placccebo) [inline src/main/scala/com/bwsw/cloudstack/vault/server/Components.scala:86]: closeFunctionList.foreach(func => {
      Try(func()) match {
        case Failure(e: Throwable) =>
          logger.error(s"the function: '$func' was executed with exception: $e")
      }
    })

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/main/scala/com/bwsw/cloudstack/vault/server/Components.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/main/scala/com/bwsw/cloudstack/vault/server/Components.scala:86]: ok

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**